api/segment_membership/mappers.py

```
@@ -1,18 +1,21 @@
-from datetime import datetime
 from decimal import Decimal
 
 from flagsmith_schemas import dynamodb
 
+# (environment_id, identifier, identity_key, traits, is_deleted)
+ClickHouseIdentityRow = tuple[str, str, str, dict[str, object] | None, bool]
 from segment_membership.constants import INT64_MAX, INT64_MIN
```

Ruff (0.15.21)

[error] Module level import not at top of file

(E402)